bibtexparser/bibdatabase.py

```diff
@@ -103,6 +116,48 @@ def expand_string(self, name):
         except KeyError:
             raise(UndefinedString(name))
 
+    def _add_missing_from_crossref_entry(self, entry, dependencies=set()):
+        if entry['ID'] in self._crossref_updated:
+            return
+
+        if entry['_crossref'] not in self._entries_dict:
```

Comment:
See comment in `_add_missing_from_crossref` about using the attribute `_entries_dict` here.

Reply:
Fixed in 74ae491ca8af740458f90ac049d25c2a17c49a95.